curriculum/test/test-challenges.js

```diff
@@ -234,28 +235,7 @@ async function getChallenges(lang) {
   );
   // This matches the order Gatsby uses (via a GraphQL query). Ideally both
   // should be sourced and sorted using a single query, but we're not there yet.
-  challenges.sort((a, b) => {
-    if (a.superOrder < b.superOrder) {
-      return -1;
-    }
-    if (a.superOrder > b.superOrder) {
-      return 1;
-    }
-    if (a.order < b.order) {
-      return -1;
-    }
-    if (a.order > b.order) {
-      return 1;
-    }
-    if (a.challengeOrder < b.challengeOrder) {
-      return -1;
-    }
-    if (a.challengeOrder > b.challengeOrder) {
-      return 1;
-    }
-    return 0;
-  });
```

Comment:
I was thinking you were going to shorten the above code to:
```suggestion
challenges.sort((a, b) => {
  return (a.superOrder - b.superOrder || a.order - b.order || a.challengeOrder - b.challengeOrder);
});
```

Reply:
I was, but I noticed lodash had a function that does that and is even shorter.  Since it passes the test and is even easier to read I thought we might as well use it.

Reply:
Got it.